### ISSUE TITLE
Commands::Deploy Error: RuntimeError

### ISSUE THREAD

**Issue by kazooligan1982** [DeployGate/deploygate-cli#36]
# Status
deploygate-cli ver 0.0.4

# Error message
[31mError packaging up the application[0m

# Backtrace
```
/Library/Ruby/Gems/2.0.0/gems/gym-1.0.0/lib/gym/error_handler.rb:86:in `handle_package_error'
/Library/Ruby/Gems/2.0.0/gems/gym-1.0.0/lib/gym/runner.rb:102:in `block in package_app'
/Library/Ruby/Gems/2.0.0/gems/fastlane_core-0.26.3/lib/fastlane_core/command_executor.rb:62:in `call'
/Library/Ruby/Gems/2.0.0/gems/fastlane_core-0.26.3/lib/fastlane_core/command_executor.rb:62:in `execute'
/Library/Ruby/Gems/2.0.0/gems/gym-1.0.0/lib/gym/runner.rb:98:in `package_app'
/Library/Ruby/Gems/2.0.0/gems/gym-1.0.0/lib/gym/runner.rb:16:in `run'
/Library/Ruby/Gems/2.0.0/gems/gym-1.0.0/lib/gym/manager.rb:8:in `work'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/builds/ios.rb:27:in `build'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/commands/deploy/build.rb:58:in `ios'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/commands/deploy/build.rb:17:in `run'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/commands/deploy.rb:16:in `run'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/command_builder.rb:37:in `block (2 levels) in run'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/runner.rb:71:in `run!'
/Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/lib/deploygate/command_builder.rb:59:in `run'
/Library/Ruby/Gems/2.0.0/gems/deploygate-0.0.4/bin/dg:6:in `<top (required)>'
/usr/bin/dg:23:in `load'
/usr/bin/dg:23:in `<main>'
```

```
error: /usr/bin/codesign --force --preserve-metadata=identifier,entitlements --sign xxx --entitlements /var/folders/dd/t61h88b94zl2yz_f1hcg79t40000gn/T/QKLYrjMAwU/entitlements_plistLsa6b9sz /var/folders/dd/t61h88b94zl2yz_f1hcg79t40000gn/T/QKLYrjMAwU/Payload/xxx.app failed with error 1. Output: iPhone Distribution: xxx (xxx): ambiguous (matches "iPhone Distribution: xxx (xxx)" and "iPhone Distribution: xxx (xxx)" in /Users/xxx/Library/Keychains/login.keychain)
```

**Comment by henteko** [comment]
memo: local install multi certificates

**Comment by henteko** [comment]
fix #43 
release v0.0.5
